OpenRA.Game/Server/Connection.cs

```diff
@@ -115,7 +115,9 @@ void SendReceiveLoop(object s)
 									frame = BitConverter.ToInt32(bytes, 4);
 									state = ReceiveState.Data;
 
-									if (expectLength < 0 || expectLength > MaxOrderLength)
+									// HACK: Disable length limit for local games (i.e. skirmish) to avoid false positive
```

Comment:
```suggestion
									// HACK: Disable length limit for local games (i.e. skirmish) to avoid false positives
```

Tbh this doesn't seem like a hack. I think it's normal and even expected for multiplayer safeguards to be disabled in singleplayer